pandas/tests/window/test_groupby.py

```diff
@@ -1040,13 +1038,10 @@ def test_times_vs_apply(self, times_frame):
         with tm.assert_produces_warning(FutureWarning, match="nuisance"):
             # GH#42738
             result = times_frame.groupby("A").ewm(halflife=halflife, times="C").mean()
-            expected = (
-                times_frame.groupby("A")
-                .apply(lambda x: x.ewm(halflife=halflife, times="C").mean())
-                .iloc[[0, 3, 6, 9, 1, 4, 7, 2, 5, 8]]
-                .reset_index(drop=True)
```

Comment:
Previously apply would call reindex, returning the groups out of order. Now the order is preserved, the call to iloc is no long necessary.